README.md

```diff
@@ -1,21 +1,28 @@
-## Stream Example App
+## Stream Django Example App
 
-This example Django app shows you how you can use [GetStream.io](https://getstream.io/ "GetStream.io") to built a site similar to Pinterest.
+This example Django app shows you how you can easily build a site similar to Pinterest.
 
-### Demo
+The project is based on the [stream-django](https://github.com/GetStream/stream-django) integration for [Stream](https://getstream.io/). There is also a lower level [Python - Stream integration](https://github.com/getstream/stream-python) library which is suitable for all Python applications.
 
-You can [try the demo here](http://exampledjango.getstream.io).
-Alternatively you can deploy your own copy of the demo via Heroku.
+You can sign up for a Stream account at https://getstream.io/get_started.
+
+If you're looking to self-host your feed solution we suggest the open source <a href="https://github.com/tschellenbach/Stream-Framework" class="mktg-text-links">Stream-Framework</a>, created by the Stream founders.
```

Comment:
change this to `[]()` markdown syntax pls ;-)

Reply:
`cmd + K` <- protip

Reply:
Done.